sidecred.go

```diff
@@ -35,10 +35,10 @@ func (r *Request) hasValidCredentials(resource *Resource) bool {
 	if r.Name != resource.ID {
 		return false
 	}
-	if !bytes.Equal(r.Config, resource.Config) {
+	if !areConfigsEqual(r.Config, resource.Config) {
 		return false
 	}
-	if resource.Expiration.Before(time.Now()) {
+	if resource.Expiration.Before(time.Now().UTC()) {
```

Comment:
I don't think this is needed since `time.Equal`, `Before` etc ignores/accounts for timezones when doing comparisons. See a playground example here: https://play.golang.org/p/iC5hwqTOSTX - or did I misunderstand something here?

Reply:
I should have read the docs more closely. Fix coming.